src/bundle/Resources/config/services.yml

```diff
@@ -1,9 +1,17 @@
 parameters:
     ezplatform.cron.registry.cronjobs.class: EzSystems\EzPlatformCronBundle\Registry\CronJobsRegistry
+    ezplatform.console.command.run.class: EzSystems\EzPlatformCronBundle\Command\CronRunCommand
```

Comment:
`*.class` parameters are deprecated since Symfony 3.4 AFAIR. Introducing new ones is a technical debt at this point. A class should be "inlined" instead.

```suggestion
```